src/pathfinding_service/schema.sql

```diff
@@ -52,5 +52,6 @@ CREATE TABLE capacity_update (
 CREATE TABLE feedback_token (
     token_id CHAR(32) NOT NULL,
     creation_time TIMESTAMP NOT NULL,
+    token_network_address CHAR(42) NOT NULL,
```

Comment:
We won't be able to make sure that the client only provides feedback for the routes we've sent to him unless we save those routes. But we can do that at a later time.